scripts/README.md

```diff
@@ -0,0 +1,426 @@
+
++ bind
+    Bind the services the application depends on, update the environment
+    variables and restage the app.
+
++ initial
+    There is no app with this name, and the services need to be bound to it
+    for it to work. the app will fail to start once, have the services bind,
+    and then get restaged.
+
++ rebuild
+    You want to redeploy the instance under the same name
+    Delete the existing app (with out deleting the services)
+    and perform the initial deployment strategy.
+
+The application name  defined via the manifest yml for the frontend
+APP_NAME=${2}
+
+### Description
+Mostly for use in our CircleCI pipelines, this script ensures the desired codebase is deployed into a given Cloud.gov application space.
+
+### Where it's used
+
+`deploy-backend.sh` is used in [CircleCI config](../.circleci/config.yml)'s job `deploy-backend` which is used by `deploy-cloud-dot-gov` command. This script is primarily via in CircleCI but with CloudFoundry set up and logged in locally, this can also be used by a developer manually. Please also see our [CircleCI documentation](../docs/Technical-Documentation/circle-ci.md#deploy-backend).
+
+## deploy-frontend.sh
+
+
+### Usage
+
+```
+./scripts/deploy-frontend.sh <strategy> <app-name>
+./scripts/deploy-frontend.sh rolling raft-review
+```
+### Arguments
+
+```bash
+DEPLOY_STRATEGY=${1}
+
+# The application name  defined via the manifest yml for the frontend
+CGHOSTNAME_FRONTEND=${2}
+CGHOSTNAME_BACKEND=${3}
+
+REACT_APP_BACKEND_HOST= #Environment Variable
+REACT_APP_BACKEND_URL= #Environment Variable
+```
+
+### Description
+Mostly for use in our CircleCI pipelines, this script ensures the desired codebase is deployed into a given Cloud.gov application space.
+
+### Where it's used
+
+`deploy-frontend.sh` is used in [CircleCI config](../.circleci/config.yml)'s job `deploy-frontend` which is used by `deploy-cloud-dot-gov` command. This script is primarily via in CircleCI but with CloudFoundry set up and logged in locally, this can also be used by a developer manually. Please also see our [CircleCI documentation](../docs/Technical-Documentation/circle-ci.md#deploy-frontend).
+
+## deploy-infrastructure-dev.sh
+
+### Usage
+
+```
+scripts/deploy-infrastructure-dev.sh
+```
+
+### Arguments
+
+#### CF Service Keys
+
+ CF_USERNAME_DEV
+ CF_PASSWORD_DEV
+
+### Description
+This script will manually run CircleCI's job `deploy-infrastructure-dev` which deploys the default Terraform codebase to a desired Cloud.gov application name in the tanf-dev space. Please see [Terraform docs](../terraform/README.md) for more information.
+
+Requires local client to have CloudFoundry and CircleCI CLI tools to be configured and logged in.
+
+Requires installation of jq - https://stedolan.github.io/jq/download/
+
+### Where it's used
+This script is no longer used by other automation as it has been obsolesced by our Terraform flows.
```

Comment:
above we say that its in use and here it sounds like maybe it isnt? can we confirm which is the case? assuming it is still in-use, can we reframe here? @riatzukiza @abottoms-coder

Reply:
I think @jorgegonzalez Could clarify this a little better than I could. Let me get  back to you on this if he doesn't. Sorry I didn't see it before the break.

Reply:
I think basicly, this is a dev tool more than an ops tool. I think maybe it saying it is no longer used by any automation is misleading, I don't think this was ever used by automation.